ingress/src/main/java/com/redhat/developer/ingress/IngressServiceImpl.java

```diff
@@ -0,0 +1,55 @@
+package com.redhat.developer.ingress;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import com.redhat.developer.infra.utils.CloudEventUtils;
+import com.redhat.developer.ingress.api.exceptions.IngressRuntimeException;
+import com.redhat.developer.ingress.producer.KafkaEventPublisher;
+
+import io.cloudevents.CloudEvent;
+
+@ApplicationScoped
+public class IngressServiceImpl implements IngressService {
+
+    private static final URI URI_PRODUCER = URI.create("ingreservice/IngressServiceImpl");
```

Comment:
Small typo `ingreservice` I think should be `ingressservice` or `ingress_service` perhaps for readability.